R/load.R

```diff
@@ -70,7 +70,8 @@ load <- function(project = NULL, quiet = FALSE, profile = NULL, ...) {
 
   action <- renv_load_action(project)
   if (action[[1L]] == "cancel") {
-    cancel()
+    autoloading <- getOption("renv.autoloader.running", default = FALSE)
+    cancel(verbose = !autoloading)
```

Comment:
Allows us to see the message after overt cancelation by the user, but suppress it when deferring the project load to the session init hook. As discussed elsewhere.